radix-engine-tests/tests/application/static_resource_movements_visitor.rs

```diff
@@ -996,26 +996,6 @@ fn aggregation_balance_change_test_cases() {
     }
 }
 
-#[test]
-fn static_analysis_on_a_vault_direct_method_succeeds() {
```

Comment:
Why was this removed out of interest? I guess because it's pointless now, because it couldn't fail to be analyzed?